module-audio/Audio/Audio.cpp

```diff
@@ -34,15 +34,7 @@ namespace audio
 
     audio::RetCode Audio::SetOutputVolume(Volume vol)
     {
-        auto volToSet = vol;
-        if (vol > maxVolume) {
-            volToSet = maxVolume;
-        }
-        if (vol < minVolume) {
-            volToSet = minVolume;
-        }
```

Comment:
imo additional security won't be a problem - consider using std::clamp

Reply:
This creates an issue after changing number of volume levels for Harmony - this code is placed in common part and it's not so simple to determine which maximum value should be used. We've checked that every call to this function is already guarded with `std::clamp`